tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -942,6 +942,9 @@ MetroHash::Hash PipelineDumper::generateHashForComputePipeline(const ComputePipe
   hasher.Update(pipeline->options.extendedRobustness.robustImageAccess);
   hasher.Update(pipeline->options.extendedRobustness.nullDescriptor);
 
+  // Relocatable shaders force an unlinked compilation.
+  hasher.Update(pipeline->unlinked || isRelocatableShader);
```

Comment:
Instead, would it be possible the update the `pipeline` info struct and always set `unliked` to true when reloc compilation is enabled?